.github/ISSUE_TEMPLATE/epic-spec.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Epic spec
+about: Epic level spec
+
+---
+
+## Overview
+Describe the output of epic
+
+## Who is it for? Who are the users?
+
```

Comment:
Add Data privacy assessment to this. See https://ico.org.uk/for-organisations/guide-to-the-general-data-protection-regulation-gdpr/accountability-and-governance/data-protection-impact-assessments/ for reference